src/main/java/com/google/cloud/genomics/utils/Paginator.java

```diff
@@ -94,9 +94,9 @@
  *String datasetId = ...;
  *Paginator.Readsets searchReadsets = Paginator.Readsets.create(stub);
  *for (Readset readset =
- *    searchReadsets.search(new SearchReadsetsRequest().setDatasetId(datasetId))) {
+ *    searchReadsets.search(new SearchReadsetsRequest().setDatasetId(datasetId))) &#123;
  *  // do something with readset
- *}
+ *&#125;
```

Comment:
did we mean to do this? Is it necessary for something? It certainly makes the comment harder to read in the IDE.